configuration.json.example

```diff
@@ -12,7 +12,9 @@
     "Password": "",
     "Prefix": "umati/v2",
     "ClientId": "MyCompany/ClientName",
-    "Protocol": "tcp"
+    "Protocol": "tcp",
+    "HttpProxy": "",
```

Comment:
Should add documentation on when and which format

Reply:
resolved by [074fb12](https://github.com/umati/Dashboard-OPCUA-Client/pull/591/commits/074fb127b99edc71bcb8c6a800ec8ba4bb40546c)